src/js/Tabs.js

```diff
@@ -214,25 +216,32 @@ function Tabs(rootEl, config) {
 	init();
 }
 
-Tabs.init = function(el, config) {
-	const tabs = [];
-	let tEls;
-	let c;
-	let l;
-	if (!el) {
-		el = document.body;
-	} else if (!(el instanceof HTMLElement)) {
-		el = document.querySelector(el);
-	}
-	if (el.querySelectorAll) {
-		tEls = el.querySelectorAll('[data-o-component=o-tabs]');
-		for (c = 0, l = tEls.length; c < l; c++) {
-			if (!tEls[c].matches('[data-o-tabs-autoconstruct=false]') && !tEls[c].hasAttribute('data-o-tabs--js')) {
-				tabs.push(new Tabs(tEls[c], config));
-			}
+Tabs.init = function(rootEl, config) {
+	if (!rootEl) {
+		rootEl = document.body;
+	}
+	if (!(rootEl instanceof HTMLElement)) {
+		rootEl = document.querySelector(rootEl);
+	}
+
+	if (rootEl instanceof HTMLElement && /\bo-tabs\b/.test(rootEl.getAttribute('data-o-component'))) {
+		if (!rootEl.matches('[data-o-tabs-autoconstruct=false]') && !rootEl.hasAttribute('data-o-tabs--js')) {
+			return new Tabs(rootEl, config);
 		}
 	}
-	return tabs;
+
+	if (rootEl.querySelectorAll) {
+		let tabs = [];
+		const tabEls = Array.from(rootEl.querySelectorAll('[data-o-component=o-tabs]'));
+
+		tabEls.forEach(tabEl => {
```

Comment:
The selector is rather large, I wonder why we have so many attributes to search for. Should o-component-boilerplate add these data attributes to it's css-selector? -- https://github.com/Financial-Times/o-component-boilerplate/blob/master/src/js/componentBoilerplate.js#L18